### DISCUSSION TITLE
Update the official tutorial, "Groundwork" part

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2521] [Ideas]
Let me explain myself

If following the tutorial, when reaching this part
![image](https://user-images.githubusercontent.com/60806448/158607234-2ca3afbe-3de7-462f-874c-7c9051676695.png)
this will cause a bug, in which player will constantly move because it's telling to move the player every frame with no condition

It can be easily fixed tho, in 2 ways i know
1: Setting the speed to something only when player is moving and setting it to 0 when player is not
![image](https://user-images.githubusercontent.com/60806448/158607675-9f708e01-b8a1-47ec-9d82-34f830a2e80a.png)
![image](https://user-images.githubusercontent.com/60806448/158607724-1ee4898b-ec28-4f20-9570-de881beb139d.png)

2: Adding a boolean that will tell if player is actually moving and move the player only if any of the keys is actually pressed
![image](https://user-images.githubusercontent.com/60806448/158608020-b5bc30c9-8bc3-4367-b203-5634ce502e93.png)
![image](https://user-images.githubusercontent.com/60806448/158608072-bb70b2cb-c8c8-47e0-9c97-437d97dafedd.png)

It really is easy, why not update the tutorial (if the reason that it IS easy, then why letting an intentional bug be in an OFFICIAL tutorial?)

**Comment by Geokureli** [comment]
yeah this bug irritates me as well. I need to look into how to even update that thing

**Reply by RichardBray** [reply]
If it's part of the main website maybe it can be added as an issue here>

https://github.com/HaxeFlixel/haxeflixel.com/issues
